pom.xml

```diff
@@ -10,12 +10,12 @@
         <groupId>org.alfresco</groupId>
         <artifactId>alfresco-enterprise-repo</artifactId>
         <relativePath>../alfresco-enterprise-repo/pom.xml</relativePath>
-        <version>8.423</version>
+        <version>8.422-SNAPSHOT</version>
```

Comment:
can you remove those version changes? Thanks

Reply:
Oh - good reminder. I have to include them so that the build passes, but once the feature branch is finalised then I need to revert this before merging.